src/herder/test/UpgradesTests.cpp

```diff
@@ -1406,6 +1406,67 @@ TEST_CASE("upgrade to version 10", "[upgrades]")
     }
 }
 
+TEST_CASE("upgrade to version 11", "[upgrades]")
```

Comment:
this is the only change in this file that I was expecting

Reply:
fixed.